### ISSUE TITLE
Would It Be Possible To Log Character Sheet Changes?

### ISSUE THREAD

**Issue by SasugaTV** [ben/foundry-ironsworn#134]
Would it be possible to log in the chat, automatically, when things like Momentum are changed, health, etc. so that I could review the log and see which choices I made with what rolls, to recreate my story later, if I so choose..?

Knowing if I choose momentum or health after rolling for camp, for example, would help me. Did my character heal up, or feel a stronger sense of purpose after resting?

I am keeping a log and journal right now, it would just be helpful it the log was automatic.

**Comment by SasugaTV** [comment]
Wow, awesome, thanks!

**Comment by ben** [comment]
Shipped!

**Comment by SasugaTV** [comment]
I hope writing this doesn't re-open the issue.

I wanted to say, this logging has come in as a lot more useful than I had originally imagined.

I am now playing Ironsworn with three other friends, and it can be difficult to keep track of things that happened as we're figuring things out... And it's just so useful.

For example: One friend forgot to mark an extra progress when they should have, thanks to the log we were able to confirm the mistake was made instead of simply wondering about whether or not it had and having to do math or whatever and try to figure it out, without ever being certain.

It's also handy for things like sharing group supply, as we've accidentally had someone add more supply not realizing others had done it, and the log is how we noticed it had happened.

I can list lots more, it's probably helped us about 20 times in this last session we just had tonight.

Thanks again!
